scripts/spcgeonode/docker-compose.yml

```diff
@@ -22,10 +22,25 @@ x-common-django:
     - TIME_ZONE=${TIME_ZONE}
     - ALLOWED_HOSTS=['nginx','127.0.0.1','localhost','$HTTPS_HOST','$HTTP_HOST']
     - SECRET_KEY=${SECRET_KEY}
+
+    ## UNCOMMENT FOR SSL
+    #- GEOSERVER_PUBLIC_LOCATION=https://${HTTPS_HOST}/gs/
+    #- GEOSERVER_WEB_UI_LOCATION=https://${HTTPS_HOST}/geoserver/
+    #- SITEURL=https://${HTTPS_HOST}
+    ## COMMENT FOR SSL
+    - GEOSERVER_PUBLIC_LOCATION=http://${HTTP_HOST}/gs/
+    - GEOSERVER_WEB_UI_LOCATION=http://${HTTP_HOST}/geoserver/
+    - SITEURL=http://${HTTP_HOST}
+
+    # database info, ${variables} are read from .env
+    - DATABASE_URL=postgres://${GEONODE_DATABASE}:${GEONODE_DATABASE_PASSWORD}@postgres:5432/${GEONODE_DATABASE}
+    - GEODATABASE_URL=postgis://${GEONODE_GEODATABASE}:${GEONODE_GEODATABASE_PASSWORD}@postgres:5432/${GEONODE_GEODATABASE}
+
     # hardcoded
+    - DEFAULT_BACKEND_UPLOADER=geonode.importer
+    - DEFAULT_BACKEND_DATASTORE=datastore

```

Comment:
What about a comment for those who want to store the layers on files?

Reply:
Was added as comment to the compose file